test_cases.md

```diff
@@ -0,0 +1,71 @@
+# Budget test cases
+
+### Unit tests
+---
+
+#### **TC-01**
```

Comment:
Numery TC-01 TC-02 tylko do testów funkcjonalnych. 

Całość nazwałabym Budgest Test Smoke Suite.

Test Suite consists of unit and functional tests.


Unit tests are in the file... 
They are covering (...)